daemon/api_model.go

```diff
@@ -69,3 +89,90 @@ func postModel(c *Command, r *http.Request, _ *auth.UserState) Response {
 	return AsyncResponse(nil, &Meta{Change: chg.ID()})
 
 }
+
+// helper for parsing url options into formatting option vars
+func parseHeadersFormatOptionsFromURL(url *url.URL) (
+	jsonResult bool,
```

Comment:
please don't format function return values like this. If they don't fit in one line you're doing something wrong :-)

Reply:
Fixed